packages/framework/fluid-framework/src/legacy.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable import/no-internal-modules */
```

Comment:
This is temporary until a lint config update? Worth noting that.

This also made me realize our lint rule preventing ./alpha imports internally needs to account for ./legacy.

Reply:
I added comments to the eslint-disable.

@jason-ha was against adding exemptions for /beta and /alpha (now /legacy) to the global lint config, arguing that 99% of the time people should import from '/internal'.